apollo-federation/src/query_graph/condition_resolver.rs

```diff
@@ -54,19 +54,6 @@ impl ConditionResolution {
 //            But in Rust, `QueryPlanningTraversal` implements `ConditionResolver` trait itself,
 //            using `ConditionResolverCache` struct below.
 //            However, `CachingConditionResolver` may still be used in the composition in the future.
```

Comment:
We should remove the comment then, or move it to the top of the file ("This file doesn't implement CachingConditionResolver, because ...")